gitops/plnsvc/base/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://raw.githubusercontent.com/roming22/pipelines-service/my/gitops/gitops/plnsvc/base/namespace.yaml
+  - https://raw.githubusercontent.com/roming22/pipelines-service/my/gitops/gitops/plnsvc/base/kcp-manager.yaml
```

Comment:
I am not following the intent. Some description would be welcome

Reply:
The description has been added to the Application definition.
The reference to an external repository was an oversight on my part.